Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   # gem 'beaker', *location_for(ENV['BEAKER_LOCATION'] || '~> 2.2')
   # See: SERVER-435 for the reasoning behind this specific version
-  gem 'beaker', *location_for(ENV['BEAKER_LOCATION'] || 'git://github.com/puppetlabs/beaker#7d0cccf22cc7289113de862fe4a4494872f574bb')
+  gem 'beaker', *location_for(ENV['BEAKER_LOCATION'] || 'git://github.com/puppetlabs/beaker#beaker2.7.1')
```

Comment:
In theory, this should be identical to using a release version of beaker by pinning to the exact version, but I'm guessing you're asking about using rubygems to install the dependency rather than bundler + git to install the dependency.

I've updated the change set to use the released gem, please let me know if that doesn't fully address the concern.